cni-plugin/internal/pkg/testutils/utils_linux.go

```diff
@@ -249,7 +249,10 @@ func RunCNIPluginWithId(
 	var r types.Result
 	pluginPath := fmt.Sprintf("%s/%s", os.Getenv("BIN"), os.Getenv("PLUGIN"))
 	r, err = invoke.ExecPluginWithResult(context.Background(), pluginPath, []byte(netconf), args, customExec)
+	//TODO: error
```

Comment:
Did you mean to leave this TODO in?

Reply:
No, might forgot to remove that. I'll see is there any other left and remove them.